src/server.zig

```diff
@@ -36,13 +36,17 @@ pub fn init(a: Allocator, router: Router, opts: Options) !Server {
     if (opts.stats) |so| {
         stats_.options = so;
     }
+
+    // TODO FIXME
+    const now: std.Io.Timestamp = .{ .nanoseconds = 1762107590 * std.time.ns_per_s };
```

Comment:
Shouldn't the Io interface be passed in as a parameter to this function to allow the user to choose their own implementation?

Reply:
The Io interface is still created inside the verse instead of being passed in.